content/docs/command-reference/exp/init.md

```diff
@@ -1,6 +1,7 @@
 # exp init
 
-Codify project using [DVC metafiles](/doc/user-guide/project-structure) to run
```

Comment:
sorry folks for going into review on top of review, it's way too complicated to my mind :)

(this change itself is not substantial enough to be honest)

Codify ... DVC metafiles .... experiments ... an existing project - I'm afraid that people won't understand this. And this is part of the happy path.

What does this command essentially do?

`The fastest way to initialize needed stuff in a project for DVC to be able to run experiments`

why don't we write like this? the current summary should be part of the description (but description starts nicely now, so I would not touch it)

Reply:
Note that any changes should also apply to docs/command-reference/exp/index.md.

Reply:
> going into review on top of review

No problem, that's kind of the intention of this PR 🙂

Reply:
> What does this command essentially do?

How about `Quickly setup any project to use [DVC Experiments].`?

Reply:
> Quickly setup any project to use [DVC Experiments]

sounds good to me!